src/Bicep.Core.Samples/Files/InvalidParameters_LF/Completions/stringParameterDecorators.json

```diff
@@ -0,0 +1,101 @@
+[
+  {
+    "label": "allowed",
+    "kind": "function",
+    "detail": "allowed()",
+    "deprecated": false,
+    "preselect": false,
+    "sortText": "3_allowed",
+    "insertTextFormat": "snippet",
+    "insertTextMode": "asIs",
+    "textEdit": {
+      "range": {},
+      "newText": "allowed($0)"
+    },
+    "command": {
+      "command": "editor.action.triggerParameterHints"
+    }
+  },
+  {
+    "label": "description",
+    "kind": "function",
+    "detail": "description()",
+    "deprecated": false,
+    "preselect": false,
+    "sortText": "3_description",
+    "insertTextFormat": "snippet",
+    "insertTextMode": "asIs",
+    "textEdit": {
+      "range": {},
+      "newText": "description($0)"
+    },
+    "command": {
+      "command": "editor.action.triggerParameterHints"
+    }
+  },
+  {
+    "label": "maxLength",
+    "kind": "function",
+    "detail": "maxLength()",
+    "deprecated": false,
+    "preselect": false,
+    "sortText": "3_maxLength",
+    "insertTextFormat": "snippet",
+    "insertTextMode": "asIs",
+    "textEdit": {
+      "range": {},
+      "newText": "maxLength($0)"
+    },
+    "command": {
+      "command": "editor.action.triggerParameterHints"
+    }
+  },
+  {
+    "label": "metadata",

```

Comment:
>metadata [](start = 14, length = 8)

Unrelated to this change, but reminded me to ask this question: What happens if you use the @description and @metadata operator with an object that has a conflicting description? Do we merge or give an error?

Reply:
In our current implementation, the one defined at the top takes precedence (there's a test for it). This also applies to the case where there are duplicate decorators.